src/features/dead-code/deadCodeConfig.ts

```
@@ -0,0 +1,53 @@
+        const config = vscode.workspace.getConfiguration(this.CONFIG_SECTION);
+        await config.update(key, value, vscode.ConfigurationTarget.Workspace);
+    }
+
+    static watchConfigChanges(callback: (config: DeadCodeConfig) => void): vscode.Disposable {
+        return vscode.workspace.onDidChangeConfiguration(e => {
+            if (e.affectsConfiguration(this.CONFIG_SECTION)) {
+                callback(this.getConfig());
+            }
+        });
+    }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)